packages/vscode-xml-tools-client/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "vscode-xml-tools-client",
+  "description": "VSCode client part of an xml language server",
```

Comment:
description should be written as facing consumer from VSCode marketplace.
basically not in technical terms